source/content/overview/index.md

```diff
@@ -0,0 +1,43 @@
+# Overview
+
+This section gives an overview on *why* and *how* the Brightway life cycle assessment (LCA) framework works, as well as some generic LCA methodology and implementation discussion. It is not designed for people learning about Brightway for the first time - please [see the Brightway book](https://learn.brightway.dev/) which is tailored for newcomers.
+
+```{note}
+If you have questions that the documentation cannot answer, you can [get in touch with the Brightway community or the Brightway developers](../contact/contact.md).
+```
+
+```{toctree}
+---
+hidden: true
+maxdepth: 2
+---
+self
+core
+inventory
+matrix
+io
+```
+
+## Brightway philosophy
```

Comment:
Hmm... I will re-capitalize all nouns in section headings when you are not looking 🙈